src/Security/MemberAuthenticator/CookieAuthenticationHandler.php

```diff
@@ -176,16 +176,20 @@ public function authenticateRequest(HTTPRequest $request)
 
         // Renew the token
         $rememberLoginHash->renew();
```

Comment:
```suggestion
        Deprecation::withNoReplacement(fn() => $rememberLoginHash->renew());
```
This is required so that the deprecation notice isn't emitted every time core calls the method. We only want the deprecation notice emitted when project code calls it, so people know to stop calling that method for their own customisations.